projects/packages/post-list/src/style.css

```diff
@@ -8,6 +8,7 @@
 	width: 50px;
 	height: 50px;
 	border-radius: 4px;
+	object-fit: cover;
```

Comment:
Just wanna say I  didn't know this existed really cool will add it to my CSS note, also It seems you are correct and we don't support IE. (PCYsg-2bz-p2)

Reply:
same :-)

Reply:
Interestingly, we do support IE in the Classic Dashboard, which is what this affects. It's probably a visual glitch that can be lived with for now.